module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Remove-EntraScopedRoleMembership.md

```diff
@@ -27,7 +27,7 @@ Removes a scoped role membership.
 
 ```powershell
 Remove-EntraScopedRoleMembership 
- -ObjectId <String> 
+ -AdministrativeUnitId <String> 
  -ScopedRoleMembershipId <String>
  [<CommonParameters>]
 ```
```

Comment:
Update the description part to reflect AdministrativeUnitId instead of ObjectId.